usage-examples/test-execution.gliffy

```diff
@@ -0,0 +1 @@
6,"lockAspectRatio":false,"lockShape":false,"order":"auto","graphic":{"type":"Text","Text":{"tid":null,"valign":"middle","overflow":"both","vposition":"none","hposition":"none","html":"<p style=\"text-align:center;\"><span style=\"font-size: 14px; font-family: Arial; white-space: pre-wrap; text-decoration: none; line-height: 16.5px; color: rgb(0, 0, 0);\">ACTIVITY_EXECUTION</span></p>","paddingLeft":2,"paddingRight":2,"paddingBottom":2,"paddingTop":2}},"children":null}],"linkMap":[]},{"x":871.5,"y":204,"rotation":0,"id":64,"uid":"com.gliffy.shape.basic.basic_v1.default.line","width":100,"height":100,"lockAspectRatio":false,"lockShape":false,"order":39,"graphic":{"type":"Line","Line":{"strokeWidth":2,"strokeColor":"#0000ff","fillColor":"none","dashStyle":null,"startArrow":0,"endArrow":17,"startArrowRotation":"auto","endArrowRotation":"auto","ortho":true,"interpolationType":"linear","cornerRadius":10,"controlPath":[[-122,-104],[-87,-104],[-87,-57],[-52,-57]],"lockSegments":{}}},"children":[{"x":0,"y":0,"rotation":0,"id":65,"uid":null,"width":67,"height":16,"lockAspectRatio":false,"lockShape":false,"order":"auto","graphic":{"type":"Text","Text":{"tid":null,"valign":"middle","overflow":"both","vposition":"none","hposition":"none","html":"<p style=\"text-align:center;\"><span style=\"font-size: 14px; font-family: Arial; white-space: pre-wrap; text-decoration: none; line-height: 16.5px; color: rgb(0, 0, 0);\">CONTEXT</span></p>","paddingLeft":2,"paddingRight":2,"paddingBottom":2,"paddingTop":2}},"children":null}],"linkMap":[]},{"x":871.25,"y":484.5,"rotation":0,"id":54,"uid":"com.gliffy.shape.basic.basic_v1.default.line","width":100,"height":100,"lockAspectRatio":false,"lockShape":false,"order":29,"graphic":{"type":"Line","Line":{"strokeWidth":2,"strokeColor":"#d5a6bd","fillColor":"none","dashStyle":null,"startArrow":0,"endArrow":17,"startArrowRotation":"auto","endArrowRotation":"auto","ortho":true,"interpolationType":"linear","cornerRadius":10,"controlPath":[[-122,-104],[-87,-104],[-87,-57],[-52,-57]],"lockSegments":{}}},"children":[{"x":0,"y":0,"rotation":0,"id":55,"uid":null,"width":170,"height":16,"lockAspectRatio":false,"lockShape":false,"order":"auto","graphic":{"type":"Text","Text":{"tid":null,"valign":"middle","overflow":"both","vposition":"none","hposition":"none","html":"<p style=\"text-align:center;\"><span style=\"font-size: 14px; font-family: Arial; white-space: pre-wrap; text-decoration: none; line-height: 16.5px; color: rgb(0, 0, 0);\">TEST_CASE_EXECUTION</span></p>","paddingLeft":2,"paddingRight":2,"paddingBottom":2,"paddingTop":2}},"children":null}],"linkMap":[]}],"background":"#FFFFFF","width":873.25,"height":632,"maxWidth":5000,"maxHeight":5000,"nodeIndex":317,"autoFit":true,"exportBorder":false,"gridOn":true,"snapToGrid":true,"drawingGuidesOn":true,"pageBreaksOn":false,"printGridOn":false,"printPaper":"LETTER","printShrinkToFit":false,"printPortrait":true,"shapeStyles":{},"lineStyles":{"global":{}},"textStyles":{},"themeData":null}}
```

Comment:
When there are multiple links it's hard, but could you try to make the edges not overlap? :) The TERC, CONTEXT and ENVIRONMENT links from TSS 1A and TSS 1B are hard to tell apart.